kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -179,6 +179,16 @@ public EntrySet newLambdaAndEmit(Positions filePositions, JCTree.JCLambda lambda
             .addSignatureSalt("" + filePositions.getSpan(lambda)));
   }
 
+  /** Emits and returns a new {@link EntrySet} representing a static class initializer. */
+  public EntrySet newClassInitAndEmit(VName classNode) {
+    return emitAndReturn(
+        newNode(NodeKind.FUNCTION)
+            .setCorpusPath(CorpusPath.fromVName(classNode))
+            .addSignatureSalt(classNode)
+            .addSignatureSalt("cinit")
```

Comment:
any reason why "cinit" and not "clinit"? is that consistency with C++ desired?

Reply:
It's just feeding the hash so it doesn't matter much but...........